src/options.cpp

```diff
@@ -1876,6 +1876,10 @@ void options_manager::serialize(JsonOut &json) const
 
     for( size_t j = 0; j < vPages.size(); ++j ) {
         for( auto &elem : mPageItems[j] ) {
+            // Skip blanks (#18870)
```

Comment:
No need to put issue # here. Expanded description of solved issue is much better.